.github/workflows/deploy-ztp-relocatable.yml

```diff
@@ -120,6 +120,8 @@ jobs:
       matrix:
         service: ${{ fromJSON(needs.extract-spokes-list.outputs.services) }}
     steps:
-      - name: ${{ matrix.service }}
+      - name: Monitor deployment of cluster ${{ matrix.service }}
```

Comment:
@karmab Please keep the original because the idea is view on the left side the name of the spoke